spec/requests/api/v1/moves_controller_filter_spec.rb

```diff
@@ -9,12 +9,30 @@
 
   describe 'GET /moves' do
     context 'when filtering' do
-      let!(:supplier) { create :supplier }
-      let!(:location) { create :location, :with_moves, suppliers: [supplier] }
-      let!(:filtered_out_moves) { create_list :move, 10 }
-      let(:filter_params) { { filter: { supplier_id: supplier.id } } }
+      let(:move_ids) { response_json['data'].map { |move| move['id'] } }
```

Comment:
- this is used just in one test not in the entire context
- this let is used for the expectation not for the arrangement/setup